templates/base/head_style.tmpl

```diff
@@ -6,3 +6,8 @@
 {{else if ne DefaultTheme "gitea"}}
 	<link rel="stylesheet" href="{{AssetUrlPrefix}}/css/theme-{{DefaultTheme | PathEscape}}.css?v={{AssetVersion}}">
 {{end}}
+{{if .RenderResponse}}
+	{{range .RenderResponse.ExtraStyleFiles}}
+		<link rel="stylesheet" href="{{.}}">
```

Comment:
It is not right. For example, your "swaggger.css" contains many different styles.

Do not pollute the Gitea page's global style anymore.

![image](https://github.com/go-gitea/gitea/assets/2114189/aeeccf0a-34b9-4db0-ab83-fc6af56e3a97)

Reply:
Yes, as mentioned above, we need to put it into an `<iframe>` to avoid CSS pollution.